graphql_compiler/cost_estimation/filter_selectivity_utils.py

```diff
@@ -45,14 +45,13 @@ def _are_filter_fields_uniquely_indexed(filter_fields, unique_indexes):
 
 
 def _get_filter_selectivity(
-    schema_graph, lookup_class_counts, filter_info, parameters, location_name
+    schema_graph, statistics, filter_info, parameters, location_name
 ):
     """Calculate the selectivity of an individual filter at a given location.
 
     Args:
         schema_graph: SchemaGraph object
-        lookup_class_counts: function, string -> int, that accepts a class name and returns the
-                             total number of instances plus subclass instances
+        statistics: Statistics object
```

Comment:
why does this function require a statistics object? it doesn't seem to be used anywhere

Reply:
follow up: filter selectivity tests might not need an empty statistics object passed in to them either if we remove this.

OTOH if it'll serve a purpose at some later point then we can leave a note here saying that

Reply:
Answering this question is not the goal of this PR.
Luckily, there's a happy ending: once functionality for the 3rd statistic gets added, this parameter will be necessary.

Reply:
Okay, I've dug through `.pylintrc` and found this:
```
    # unused-argument,         # needs some cleanup and per-line suppression,
                               # buggy / unclear how to suppress only a single function
```